tardis/model/base.py

```diff
@@ -606,9 +606,8 @@ def from_csvy(cls, config):
             ), "CSVY field descriptions exist without corresponding csv data"
             if unsupported_columns != set():
                 logger.warning(
-                    "The following columns are specified in the csvy"
-                    "model file, but are IGNORED by TARDIS: %s"
-                    % (str(unsupported_columns))
+     f"The following columns are specified in the csvy model file, but are IGNORED by TARDIS: {str(unsupported_columns)}"
```

Comment:
Please adhere to the `Black` formatting styling. This seems to be off here in terms of the formatting of the message under the `logger.warning`